packages/lesswrong/components/messaging/ConversationPage.tsx

```diff
@@ -59,7 +68,7 @@ const ConversationPage = ({ documentId, terms, currentUser, classes }: {
     </div>
   }
 
-  if (loading) return <Loading />
+  if (loading || loadingTemplate) return <Loading />
```

Comment:
This probably adds a very slight delay to all _non_ templated messages, which is barely noticeable but is worth checking "can we not do that?"

Reply:
Yeah, you can just check for `&& query?.templateCommentId`